doc/nrf/glossary.rst

```diff
@@ -381,9 +405,15 @@ Glossary
       It was initially created for the Linux kernel.
       It uses the Kconfig configuration language.
 
+   Modulation Coding Scheme (MCS)
+      An index defined by the Wi-Fi standards where each index specifies a transmission rate that is a unique combination of various modulation schemes, forward error correction coding rates and spatial streams.
```

Comment:
This is not strictly true anymore, since the MCS in 11ac and 11ax are no longer unique across bandwidth and spatial streams. This description may have been true in 11n (Wi-Fi 4).

Reply:
Would the definition be true without the word "unique"?

Reply:
> This is not strictly true anymore, since the MCS in 11ac and 11ax are no longer unique across bandwidth and spatial streams. This description may have been true in 11n (Wi-Fi 4).

Correct, and hence I have not mentioned "bandwidth". 11n is an aberration but since this is a generic definition, I thought we should add the "spatial streams" aspect to this.

Reply:
Let's keep it concise and what it is today, i.e. -
An index defined by the Wi-Fi standards where each index specifies a combination of modulation scheme and forward error correction coding rate.

Reply:
Agree

Reply:
Fixed.